cmd/node/factory/structs.go

```diff
@@ -709,8 +708,8 @@ func newEpochStartTrigger(
 			Storage:         args.data.Store,
 			RequestHandler:  requestHandler,
 			Epoch:           args.startEpochNum,
-			Validity:        0,
-			Finality:        0,
+			Validity:        process.MetaBlockFinality,
```

Comment:
what is this field for and why is it initialized with MetaBlockFinality?

Reply:
is it used as the number of blocks that give the finality?

Reply:
added verification that metablock is valid as final as well. you must have at least validity num of blocks before the epoch announcing metablock, and finality number of blocks after.

Reply:
Yes but they are set o the same value MetaBlockFinality. Is it ok?

Reply:
done